privaterelay/templates/includes/header.html

```diff
@@ -1,5 +1,16 @@
+{% load i18n %}
 {% load static %}
+
+{% get_current_language as LANGUAGE_CODE %}
+
 <header class="flx flx-col spc-btwn">
+
+  <div class="recruitment-banner">
+    {% if settings.RECRUITMENT_BANNER_LINK and LANGUAGE_CODE|slice:"0:2" == "en" %}
+      <a id="recruitment-banner" class="text-link" href="{{ settings.RECRUITMENT_BANNER_LINK }}" target="_blank" rel="noopener noreferrer" data-ga="send-ga-funnel-pings" data-event-category="Recruitment" data-event-label="{{ settings.RECRUITMENT_BANNER_TEXT }}">{{ settings.RECRUITMENT_BANNER_TEXT }}</a>
```

Comment:
Do we need to escape the `data-event-label` attribute, like we do on https://github.com/mozilla/blurts-server/pull/1957/files#diff-d803a6e07903be9fc08d03c1e5bdc59f6513e8f385d8b482df0f0991bcf91dd3R44?

Reply:
No need - Django templates [automatically escape HTML](https://docs.djangoproject.com/en/3.1/ref/templates/language/#automatic-html-escaping).